src/main/java/org/fogbowcloud/saps/engine/core/archiver/storage/swift/SwiftAPIClient.java

```diff
@@ -76,9 +83,7 @@ public void createContainer(String containerName) {
         try {
             Process p = builder.start();
             p.waitFor();
-        } catch (IOException e) {
-            LOGGER.error("Error while creating container " + containerName, e);
-        } catch (InterruptedException e) {
+        } catch (IOException | InterruptedException e) {
```

Comment:
I believe that this method should throw an exception when it is not possible to create the container, because the Archiver component is inoperative without having a Swift container to store its data and should be logged in immediately to resolve this, which can be a simple error in the configured properties.

Reply:
Added TODO to throws exception when container creation was not success.

Reply:
out of curiosity ... isn't this important enough to implement right now?

Reply:
@wesleymonte and @thiagoyeds, why not solving this right now? it seems important to me

Reply:
@thiagomanel, I solved this problem.